server/src/main/java/org/opensearch/search/profile/Timer.java

```diff
@@ -53,6 +53,18 @@ public class Timer {
     private boolean doTiming;
     private long timing, count, lastCount, start, earliestTimerStartTime;
 
+    public Timer() {
+        this(0, 0, 0, 0, 0);
+    }
+
+    public Timer(long timing, long count, long lastCount, long start, long earliestTimerStartTime) {
```

Comment:
Don't see a need for this constructor, please remove this change

Reply:
This constructor is used to create testing timers for `testMergeRewriteTimeIntervals()` in ConcurrentQueryProfilerTests.java